docs/Using-Fleet/Fleet-UI.md

```diff
@@ -1,69 +1,63 @@
 # Fleet UI
-- [Scheduling queries](#scheduling-queries)
-- [Configuring agent options](#configuring-agent-options)
+- [Create a query](#schedule-a-query)
```

Comment:
Is this the correct destination for creating a query?

Reply:
Nope. Great catch. Thank you.

I fixed these anchor links.